x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules.ts

```diff
@@ -132,6 +132,59 @@ export default ({ getService }: FtrProviderContext) => {
         expect(bodyToCompare).to.eql(outputRule);
       });
 
+      it('should update a single rule property and remove the action', async () => {
```

Comment:
Should we also add a test for the path where `action` is `undefined`?

Reply:
The function here https://github.com/elastic/kibana/pull/117910/files/cdbe5ccd10d34dedb49b280a1175e7757eb0ae59#diff-e53b7629fb39878bc3ed1a9a6d51e742c226f50448dd67734df473e075207da2R166

Returns a rule update body where actions is not defined so we are covered 👍